app/src/pages/projects/ProjectsPage.tsx

```diff
@@ -313,8 +454,10 @@ function ProjectItem({
               level={2}
               css={css`
                 overflow: hidden;
-                white-space: nowrap;
-                text-overflow: ellipsis;
+                display: -webkit-box;
+                -webkit-box-orient: vertical;
+                -webkit-line-clamp: 2;
```

Comment:
This is tested and working in Chrome, Firefox, and Safari.

It allows us to truncate text after a specified number of lines.

![2025-05-07 14 54 13](https://github.com/user-attachments/assets/2659ece4-ce0b-4006-b198-d7dc8366d486)